spanner/mutation.go

```diff
@@ -198,6 +198,11 @@ func structToMutationParams(in interface{}) ([]string, []interface{}, error) {
 	var cols []string
 	var vals []interface{}
 	for _, f := range fields {
+		if f.ParsedTag != nil {
+			if tag, ok := f.ParsedTag.(spannerTag); ok && tag.Generated {
```

Comment:
nit: I would prefer the more generic `ReadOnly` instead of `Generated`. That makes it clearer what it is for.

Reply:
Done